saskatoon/saskatoon/.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
+/.env
+/saskatoon/.env
+saskatoon/.env
```

Comment:
we already have these in gitignore:
```
.env
.venv
env/
venv/
ENV/
env.bak/
venv.bak/
```
I believe that `.env` pattern should already ignore your `saskatoon/.env` file. Please confirm

Reply:
Yes i placed it in the wrong folder, that's why i had this issue. i've resolved mine locally thanks a lot